.github/workflows/master.yml

```diff
@@ -25,3 +45,53 @@ jobs:
         export PATH="$HOME/miniconda/bin:$PATH"
         source activate test-environment
         ./travis/run-small-python-tests.sh
+
+  java:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Install dependencies
+      run: |
+        export GITHUB_WORKFLOW=1
+        source ./travis/install-common-deps.sh
+    - name: Run tests
+      run: |
+        export GITHUB_WORKFLOW=1
+        export PATH="$HOME/miniconda/bin:$PATH"
+        source activate test-environment
+        cd mlflow/java
+        mvn clean package -q
+
+  js:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Node
+      uses: actions/setup-node@v1
```

Comment:
Looks like this currently uses node 10.x (from github actions JS [build logs](https://github.com/mlflow/mlflow/pull/2724/checks?check_run_id=595900760)):

```
Run actions/setup-node@v1
  with:
    always-auth: false
    node-version: 10.x
```


Maybe we should pin to that version? Couldn't figure out how the nodejs action picks which version to install if none is specified (looking at [this code](https://github.com/actions/setup-node/blob/339aa41a8d72adcf661974ca2a947fa1e144530e/src/setup-node.ts#L12-L18), you'd expect it to not install nodejs at all if no `version` or `node-version` attribute is specified)

Reply:
Done!